src/main/java/com/kodilla/ecommercee/User.java

```diff
@@ -0,0 +1,98 @@
+package com.kodilla.ecommercee;
+
+import com.kodilla.ecommercee.temp.Cart;
+import com.kodilla.ecommercee.temp.Order;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "users")
+public class User {
+    private long id;
+    private String name;
+    private String surname;
+    private String mail;
+    private String password;
+    private Cart cart;
+    private List<Order> order = new ArrayList<>();
+
+    public User(long id, String name, String surname, String mail, String password) {
+        this.id = id;
+        this.name = name;
+        this.surname = surname;
+        this.mail = mail;
+        this.password = password;
+    }
+
+    @Id
+    @GeneratedValue
+    @NotNull
+    @Column(name = "ID", unique = true)
```

Comment:
To można dodać przy argumentach (linia 14)

Reply:
unique = true jest zbędne

Reply:
unique = true usunięte
adnotacje zmieniona na przy polach